src/libs/actions/App.js

```diff
@@ -124,7 +139,7 @@ function openApp() {
  * Refreshes data when the app reconnects
  */
 function reconnectApp() {
-    API.read('ReconnectApp', {policyIDList}, {
+    API.read('ReconnectApp', {policyIDListExcludingWorkspacesCreatedOffline}, {
```

Comment:
The parameter expected in the Web API is `policyIDList`, not `policyIDListExcludingWorkspacesCreatedOffline` 😆

Reply:
😬

Reply:
I'll correct this in this PR btw: https://github.com/Expensify/App/pull/12129